http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -45,7 +45,7 @@ final case class AppRecord(id: AppId,
                            dateAccessed: Instant,
                            namespaceId: NamespaceId,
                            diskId: Option[DiskId],
-                           customEnvironmentVariables: Map[String, String])
+                           customEnvironmentVariables: Option[Map[String, String]])
```

Comment:
can we make remove `Option` in the data type, and only have `Option` in `def customEnvironmentVariables = column[Option[Map[String, String]]]("customEnvironmentVariables")`

Reply:
We could but then can't use `<> (AppRecord.tupled, AppRecord.unapply)`. I guess another approach is to have some data type for env vars instead of `Map[String, String]`

Reply:
mm I see...